### ISSUE TITLE
"What is a Query" - Give more context for top-level node field 

### ISSUE THREAD

**Issue by aarmand** [learnrelay/learnrelay#32]
TL:DR - good idea would be to link the concepts to source material. Think, "If some guy shows up without a clue but thought he would learn just using this, would he easily be able to find the related info(in docs, blog, etc)?"

On https://learnrelay.org/queries/what-is-a-query you talk about "node(id: $id)". I did a few searches to get a better definition. Only place I found it was on the relay docs readme here https://github.com/relay/relay. I guess the bigger issue is how much the people using this will know coming in. From talking to you guys, seems like the point of this is to get some good resources out there for people to get on board with this. We talked about how I had tried to get into it but it was super hard to follow and how this might be a solution. So I guess the idea is that maybe there are small things you can do for the less advanced dev. The reality is, people are not going to know alot about this coming in. Nodes is not something I have really come across except in Relay/GraphQL. So 'node' wasnt foreign to me but i still wanted to look it up. And I cant say I found an answer that satisfied.

**Comment by aarmand** [comment]
https://annot.io/github.com/facebook/relay

**Comment by aarmand** [comment]
I see that Nodes is defined in https://learnrelay.org/connections/what-is-a-connection a few pages later. I wonder if it wouldnt be useful to have a definitions section early on so that people will have at least seen the words before they start getting invoked, even if the more complete usage is explained later.

**Comment by marktani** [comment]
We certainly have to deal with the problem of presenting many new words and concepts to the readers.

I don't support your suggestion of a definition section early on though, as I think it will be more overwhelming than without this section. I think one strategy we can follow is to shortly explain new words/concepts in one sentence when they appear and reassure the reader that the concept will be further explained in a later chapter. I'd rather like to sprinkle new words and concepts 'on the go' like this. You really have to understand everything a little bit before you can understand everything completely in this case.

Refering to your specific issue here though, I agree that we should give more context for `node(id: $id)`.
